### PR TITLE
chore: Ignore package-lock.json

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

**Comment by DavidAnson** [comment]
I'd rather not allow this file to sit around invisibly. More info here:￼ https://github.com/DavidAnson/vscode-markdownlint/pull/136#issuecomment-735023819

**Comment by nschonni** [comment]
This keeps it ignored so it doesn't get published in the `.npmignore` file. Alternately, I think on that other PR, there was a version that used the `.npmrc` as an alternate.

**Comment by DavidAnson** [comment]
I like that it shows up as an added file when it exists so that I know to delete it. :)

**Comment by nschonni** [comment]
OK, I went with the `.npmrc` file and ignored it from the publishing. This will not produce the file with a regular `npm install`, but if it get's there somehow, it will still show up in the diff

File: .npmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+package-lock=false
```
Comment (DavidAnson) [inline .npmrc:1]: Please add newline at end.

**Review by DavidAnson — CHANGES_REQUESTED** [review]
Ok, thanks, let's try this. The commit title may need updating?

File: .npmrc (hunk 1 above)
Comment (nschonni) [reply .npmrc:1]: It's there 😉 , I think GitHub diff might not be showing it that clearly. When there is no space, you usually see the "cancel" like symbol in the diff

File: .npmignore [hunk 2]
```diff
@@ -11,3 +11,4 @@ lib-es3
 npm-debug.log
 test
 test-repos
+.npmrc
```
[the remark this answers is not in the document]
Comment (nschonni) [reply .npmignore]: Done

File: .npmrc (hunk 1 above)
Comment (nschonni) [reply .npmrc:1]: Do you want me to add an EditorConfig in a separate PR to enforce that (except for the test files)?

Comment (DavidAnson) [reply .npmrc:1]: No thanks - I'm not sold on editor config.

**Review by DavidAnson — APPROVED** [review]

**Pull request merged**